test/fuzz/yaml/yaml.go

```diff
@@ -38,6 +79,25 @@ func FuzzSigYaml(b []byte) int {
 	return out
 }
 
+// FuzzTime is a fuzz target for unmarshaling Time defined in "k8s.io/apimachinery/pkg/apis/meta/v1".
+// This target also checks that the unmarshaled result can be marshaled back to the input.
+func FuzzTime(b []byte) int {
```

Comment:
FuzzTime seems to be added between FuzzYamlV2 and FuzzSigYaml.
i think that either FuzzYamlV2 and FuzzSigYaml definitions should be adjacent or all tests that use `yaml` vs `sigyaml` should be organized.

Reply:
After updating to use sigyaml instead of yaml, all of the sigyaml targets are now grouped together (FuzzYamlV2 is at the bottom by itself). I wasn't sure what the convention is so I ordered the tests by name, but I agree that it would make sense to group the meta/v1 tests together.